instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Comment from './Comment'
+
+const CommentSection = (props) => {
```

Comment:
You can destructor props and then ...rest to pass the rest of props down. 
const CommentSection = ({comments, ...rest}) => {
    return (
        {comments.map(item => <Comment {...rest} />)}
    )
}